src/core/api/public_api.ts

```diff
@@ -2179,14 +2182,6 @@ class Player extends EventEmitter<IPublicAPIEvent> {
       });
   }
 
-  private _priv_onStreamEvent(event: IStreamEvent) {
```

Comment:
I would have had a conflict on the name `IStreamEvent` here. I preferred just removing those functions